CHANGELOG.rst

```diff
@@ -130,6 +130,10 @@ Changed
   (`#316 <https://github.com/HazyResearch/fonduer/issues/316>`_)
   (`#368 <https://github.com/HazyResearch/fonduer/pull/368>`_)
 
+Fixed
+^^^^^
+* `@senwu`_: Fix multimodal mention doesn't return mention class.
```

Comment:
Let's make this more descriptive. Is this specific to multimodal? Should it be something more general like "fix X so that Y returns mention classes, rather than data model classes"?

Reply:
After reading the code, perhaps we call this:

Fix mention extraction to operate on mention classes, not data model